Automattic/simplenote-android
# Update Style CSS

### Fix
Update the CSS file used for the **_Preview_** tab to be determined based on the style selected.  This was originally added in [Add Style Setting (Part 1)](https://github.com/Automattic/simplenote-android/pull/1116), but it must have been lost in a merge conflict somewhere.  Currently, only the **_Black_** and **_Matrix_** styles in **_Dark_** theme and the **_Sepia_** style in **_Light_** and **_Dark_** themes have custom CSS.  See the screenshots below for illustration.

![update_style_css](https://user-images.githubusercontent.com/3827611/100102273-9a732280-2e20-11eb-9c86-3e7d8138b84c.png)

I noticed links are using `simplenote_blue_50` (`#3361cc`) for **_Light_** theme and `simplenote_blue_20` (`#84a4f0`) for **_Dark_** theme on the **_Preview_** tab regardless of the selected style while the **_Edit_** tab shows links with the style's accent color.  @SylvesterWilmott, do we want the links on the **_Preview_** tab to match the links on the **_Edit_** tab?  The consequence of doing that would be that we need to have two CSS files per style; one for **_Light_** and one for **_Dark_**.  That's not technically difficult, but we'll need to maintain each of those files if we ever change the colors for styles.  I don't think that's a problem and the benefits of the style may outweigh the consequence, but I wanted to point it out for transparency.

### Test
Follow the steps below for each style by choosing a different style in Step 5 each time.
1. Open navigation drawer.
2. Tap ***Settings*** item in drawer.
3. Tap ***Style*** item under ***Appearance*** section.
4. Notice ***Style*** screen with ***Default***, ***Matrix***, ***Publication***, ***Mono***, ***Sepia***, ***Black***, and ***Classic*** styles.
5. Tap ***Default***, ***Matrix***, ***Publication***, ***Mono***, ***Sepia***, ***Black***, or ***Classic*** style.
6. Tap back arrow in app bar.
7. Tap back arrow in app bar.
8. Tap any note in list with markdown enabled.
9. Tap ***Preview*** tab.
10. Notice markdown preview is styled as shown in screenshots above.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

#### Note
Contact me for an installable build.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.
